podman_compose.py

```diff
@@ -1149,6 +1149,47 @@ def flat_deps(services, with_extends=False):
     for name, srv in services.items():
         rec_deps(services, name)
 
+###################
+# Override and reset tags
+###################
+
+class OverrideTag(yaml.YAMLObject):
+    yaml_dumper = yaml.Dumper
+    yaml_loader = yaml.SafeLoader
+    yaml_tag = u'!override'
```

Comment:
If I remember correctly, `u` prefix is from python 2 times. Am I correct that it's not needed here?